packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -873,13 +874,14 @@ export class PersonalDetailsForm extends React.Component {
                                 </FormBody>
                                 <FormFooter>
                                     {status && status.msg && <FormSubmitErrorMessage message={status.msg} />}
-                                    {!(isSubmitting || is_submit_success || (status && status.msg)) && (
-                                        <div className='account-form__footer-note'>
-                                            {localize(
-                                                'Please make sure your information is correct or it may affect your trading experience.'
-                                            )}
-                                        </div>
-                                    )}
+                                    {!this.props.is_virtual &&
+                                        !(isSubmitting || is_submit_success || (status && status.msg)) && (
+                                            <Text className='account-form__footer-note'>
```

Comment:
you should add size to the Text component and also remove the related css on the class .account-form__footer-note
```suggestion
                                            <Text className='account-form__footer-note' size='xxxs'>
```